image/walker.go

```diff
@@ -92,11 +92,13 @@ func newPathWalker(root string) walker {
 
 func (w *pathWalker) walk(f walkFunc) error {
 	return filepath.Walk(w.root, func(path string, info os.FileInfo, err error) error {
+		if info == nil {
```

Comment:
Need to check `err` here. It is the error returned by the stat call that populates this `os.FileInfo`.